src/routes/route.py

```diff
@@ -0,0 +1,32 @@
+class Route:
+    def __init__(self, path: list[str], num_people_on_route: int) -> None:
+        self.path = path
+        self.num_people_on_route = num_people_on_route
+        self.departure_times: dict[int, int] = {0: num_people_on_route}
+
+    def introduce_departure_times(self, chunks: int, interval: int) -> None:
+        """
+        Introduces different departure times for the persons taking the given route.
+
+        :param chunks: In how many different chunks should the persons on the route depart.
+        :param interval: How long should the interval between each departure be. Given in seconds.
+        :return: A dictionary of departure times where keys are seconds after midnight and values are the number of people departing on that time.
+        """
+        departure_times: dict[int, int] = {}
+        people_per_interval = int(self.num_people_on_route / chunks)
+        remaining_people = self.num_people_on_route % chunks
```

Comment:
Just a few Python tips, but you don't have to do anything:

Python has a separate `\\` operator for integer division, so this is also valid:
```suggestion
        people_per_interval = self.num_people_on_route // chunks
        remaining_people = self.num_people_on_route % chunks
```

Python also has a built-in function to do both at once:
```suggestion
        people_per_interval, remaining_people = divmod(self.num_people_on_route, chunks)
```